BEIMA.Client/src/shared/NavBar.js

```diff
@@ -0,0 +1,35 @@
+import logo from './BSU-logo.png';
+import { Row, Col} from 'react-bootstrap';
+import './shared.css';
+
+const NavBar = () => {
```

Comment:
We might consider adding ids on components so that we can uniquely identify them in testing.

Reply:
There will only be one nav bar and one page title throughout, I think in this case we don't need IDs on the nav bar and page title